src/actions/entropy.js

```diff
@@ -69,9 +69,6 @@ export const changeEntropyCdsSelection = (arg) => (dispatch, getState) => {
   if (!entropy.loaded) return;
 
   if (arg === nucleotide_gene) {
-    if (entropy.selectedCds === nucleotide_gene) {
-      return
-    }
```

Comment:
If you've highlighted a nucleotide bar (in the entropy panel) and click "reset layout" the bar will no longer be highlighted because now `action.selectedPositions = [];` will be hit below.

Reply:
Simply dropping `action.selectedPositions = []` doesn't work, I think because zoom bounds are tightly coupled with selected positions here:

https://github.com/nextstrain/auspice/blob/1f40cd0d5caaecc1b7ae47e7bb2e707b2ed92452/src/components/entropy/entropyD3.js#L131-L145